### ISSUE TITLE
How can I continue my Lora(as well as classic fine tune) training without starting it over?

### ISSUE THREAD

**Issue by terrificdm** [kohya-ss/sd-scripts#466]
Supposed I have done the Lora training, but the result wasn't as expected, a bit of under-fitting. My question is how can I continue the training basing on current Lora result without starting it over from beginning. BTW, I saved Lora as safetensor format. Should I use --resume or sth?

Another similar question for classic fine tune, regarding the same challenge, should I just change model.safetensors to diffuser format then point "pretrained_model_name_or_path" to the directory of diffusers files, and continue my training?

Thanks for help.

**Comment by RainInRiver** [comment]
I see this parameter in 5.3. May be this is the one? Does someone know how to use ?
![image](https://user-images.githubusercontent.com/117289274/236591152-9b725c2d-7f36-4870-8c74-1e75ac2f702b.png)

**Comment by terrificdm** [comment]
> I'm not familiar to GUI wrapper, but we can use `--network_weight` option to continue training from the trained model.
> 
> If you save the state with `--save_state` option, you can use `--resume` option to resume the training. `--resume` option also restores the state of the optimizer, it will be better than `--network_weight`.

Thanks for answering my question. Regarding classic fine tuning, if I want to continue my fine tuning, can I just  point  "pretrained_model_name_or_path" to the new directory of trained diffusers files? Or as you advice to use "--resume" option. I don't think there is difference  between those two options regarding output performance, am I correct? Thanks.

**Comment by terrificdm** [comment]
> One thing to keep in mind is that lr scheduler only works up to the `max_train_epochs` (or steps) specified in the initial training. If you want to stop and restart the training, specify a larger value to it.

Does that mean if I use `--save_state` + `--resume` to stop and continue my training, I still need to specify a larger value for lr? Or I don't need to worry it because `--save_state` option saves the state of the optimizer, and `--resume` can leverage it.

**Comment by terrificdm** [comment]
I see. Thank you very much for the elaboration. And great job you have done in this Repo!

**Comment by taintedsilk** [comment]
@DeJoon did you figure that out?

**Comment by DeJoon** [comment]
@taintedsilk Unfortunately not. It didn't work for me and the model that came out was just bad. Waited like ~30 hours for it across multiple days with breaks when I was home.
